Application/EdFi.Common/EdFi.Common.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net48</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net48</TargetFrameworks>
```

Comment:
Thought we were dropping to just 2.1?  Do we have a separate ticket on making this entire project only 2.1 (no 2.0 or net48).

Reply:
We are but that is another ticket. ODS-4454